configure

```diff
@@ -40,11 +40,30 @@ if (config === 'dev') {
   browserify += ' -g uglifyify'
 }
 
+function isSubmitDependency(name) {
+  const dependencies = [
+    'github-url-to-object',
+    'html-to-react',
+    'url',
+    'marky-markdown',
+    'lodash.camelcase',
+    'immutable',
+    'path'
+  ]
```

Comment:
I don't think this is the full list yet. Tracespace packages are conspicuously missing.

Reply:
Yup, missed a few. This look complete:

'github-url-to-object',
'html-to-react',
'url',
'marky-markdown',
'lodash.camelcase',
'immutable',
'path',
'whats-that-gerber',
'pcb-stackup',
'xml-element-string',
'rst2mdown'

Also, are the packages `lodash.throttle` and `gerber-to-svg` being used? I don't see a reference outside the package.json file.

Reply:
gerber-to-svg is a dependency of tracespace, we may have used it directly in the past. It may have also been in an attempt to lock down the exact version, you can probably remove it but just check we don't get an incompatible version. 

If there is no lodash.throttle anywhere then it can definitely be removed. Might be worth checking for other unused dependencies in the main code as well. Browserify can help list dependencies by the way:

```
npx browserify --extension='.jsx' --transform [ babelify  --presets [ react es2015 ] ] --list src/submit/submit.jsx
```

Reply:
Removed gerber-to-svg and everything seems to be working as expected.

The Browserify command is helpful, spits out a pretty giant list through!